modules/cli/pages/index.adoc

```diff
@@ -300,6 +300,44 @@ $ skupper service bind hello-world-backend deployment hello-world-backend
 
 --
 
+// Type: procedure
+[id="exposing-services-from-different-ns"] 
+=== Exposing services from a different namespace to the {service-network}
+
+This section shows how to expose a service from a namespace where {skupper-name} is not deployed.
+
+.Prerequisites
+
+* A namespace where {skupper-name} is deployed.
+* A service running in a different namespace.
+* A network policy that allows communication between the namespaces
+
+. Log into your cluster and change to the namespace for your site.
+
+. Expose the service on the {service-network}:
++
+--
+----
+$ skupper gateway expose service <service>.<namespace> --address <service>
```

Comment:
iiuc, you did not intend gateway here...

Reply:
doh!